pkg/registry/apis/query/query.go

```diff
@@ -150,7 +150,7 @@ func (r *queryREST) Connect(connectCtx context.Context, name string, _ runtime.O
 func (b *QueryAPIBuilder) execute(ctx context.Context, req parsedRequestInfo) (qdr *backend.QueryDataResponse, err error) {
 	switch len(req.Requests) {
 	case 0:
-		break // nothing to do
+		qdr = &backend.QueryDataResponse{}
```

Comment:
Out of curiosity who calls query without providing any queries? Shouldn't that return a 400 bad request or something instead? 🤔

Reply:
I don't think anyone does. I am good if we decide to 400 instead of responding with an empty result.